date-block.php

```diff
@@ -67,14 +65,14 @@ function date_block_init() {
 		}
 	}
 }
-add_action( 'init', __NAMESPACE__ . '\date_block_init' );
+add_action( 'init', __NAMESPACE__ . '\init' );
 
 /**
  * Registers the block textdomain.
  *
  * @return void
  */
-function date_block_i18n() {
+function i18n() {
 	load_plugin_textdomain( 'date-block', false, plugin_basename( PIXALB_DATE_BLOCK_PATH ) . '/languages' );
 }
-add_action( 'plugins_loaded', __NAMESPACE__ . '\date_block_i18n' );
+add_action( 'init', __NAMESPACE__ . '\i18n' );
```

Comment:
`load_plugin_textdomain` should not be called until `init`.
JIT! https://make.wordpress.org/core/2016/07/06/i18n-improvements-in-4-6/